### PR TITLE
[Pipeline] small changes 

### PR DESCRIPTION
**Pull request by danyaljj**
- changes in server for handling body parameters
 - adding memory requirements to the pom file of the pipeline.

### CODE REVIEW

File: pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/server/MainServer.java [hunk 1]
```diff
@@ -58,26 +62,32 @@ public static void main(String[] args) {
...
 
         post("/annotate", (request, response) -> {
-                String text = request.queryParams("text");
-                String views = request.queryParams("views");
+                logger.info("POST request . . . ");
+                logger.info( "request.body(): " + request.body());
+                Map<String, String> map = splitQuery(request.body());
+                System.out.println("POST body parameters parsed: " + map);
+                String text = map.get("text");
+                String views = map.get("views");
```
Comment (danyaljj) [inline pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/server/MainServer.java]: FYI @GHLgh this is how I changed the post request. Looks good?

Comment (GHLgh) [inline pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/server/MainServer.java]: Looks good to me. Did you try to send a post request to see if it works via Postman or similar tool?

Comment (danyaljj) [inline pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/server/MainServer.java]: Yup. 
screen shot 2017-03-05 at 5 23 44 pm

Comment (danyaljj) [inline pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/server/MainServer.java]: Deployed it, if you wanna use it: http://cogcomp.cs.illinois.edu/m2repo/edu/illinois/cs/cogcomp/illinois-nlp-pipeline/3.0.103/

**Pull request merged**